osu.Framework/Platform/Windows/WindowsClipboard.cs

```diff
@@ -23,6 +25,9 @@ public class WindowsClipboard : Clipboard
         [DllImport("user32.dll")]
         private static extern IntPtr SetClipboardData(uint uFormat, IntPtr hMem);
 
+        [DllImport("user32.dll")]
+        private static extern uint RegisterClipboardFormatW(IntPtr lpszFormat);
```

Comment:
```suggestion
        [DllImport("user32.dll", SetLastError = true)]
        private static extern uint RegisterClipboardFormatW(IntPtr lpszFormat);
```

https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-registerclipboardformatw#return-value

Reply:
I noticed that the same applies to all other imported clipboard functions, should I be setting this flag on all of them?

Reply:
Probably yes.

Reply:
Alright, will do. What is the expected behaviour here if things fail (i.e. during copying or retrieving a value)? If I throw an exception this will need to be handled on the osu side of things.

Reply:
Probably don't throw, log an error, return false to indicate clipboard wasn't set.

Reply:
I skipped `IsClipboardFormatAvailable` and `GetClipboardData` since a zero return value is expected here and just means that the format was not unavailable.

Reply:
> I skipped `IsClipboardFormatAvailable` and `GetClipboardData` since a zero return value is expected here

I mean both functions still expose error info via `GetLastError()` as far as I can tell? Even if we're not gonna check it they should be set as such.